pya/arecorder.py

```diff
@@ -108,7 +107,7 @@ def _recorder_callback(self, in_data, frame_count, time_info, flag):
             self.record_buffer.append(data_float)
             # E = 10 * np.log10(np.mean(data_float ** 2)) # energy in dB
             # os.write(1, f"\r{E}    | {self.block_cnt}".encode())
-        return None, pyaudio.paContinue
+        return self.backend.process_buffer(None)
```

Comment:
ok, maybe add a comment that `process_buffer` is called both for input and output streams